extra-devel/arduino-mk/autobuild/build

```diff
@@ -0,0 +1,23 @@
+_MAKEFILE_DIR=/usr/share/arduino
+_BIN_DIR=/usr/bin
+_MAN1_DIR=/usr/share/man/man1
+
+abinfo "Copying makefiles"
+for i in *.mk; do
+	install -Dvm644 "$i" "${PKGDIR}${_MAKEFILE_DIR}/$i"
+done
+
+abinfo "Copying exmaple programs"
+cp -vr examples -t "${PKGDIR}${_MAKEFILE_DIR}/"
+
+abinfo "Copying tools"
+pushd bin
+for i in *; do
+	install -Dvm755 "$i" "${PKGDIR}${_BIN_DIR}/$i"
+done
+popd
+
+abinfo "Copying manuals"
+for i in ard-reset-arduino.1 robotis-loader.1; do
```

Comment:
Absolute paths here.

Reply:
Since this is not a glob expression, absolute paths / relative paths are irrelavant here.